<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoaderDialog</class>
 <widget class="QWidget" name="LoaderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load…</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0">
   <item>
    <layout class="QHBoxLayout" name="rowSelAnnotationFile" stretch="0,1,0">
     <item>
      <widget class="QLabel" name="headerAnnotationFile">
       <property name="text">
        <string>Annotation File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fieldAnnotationFileUrl">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPickAnnotationFile">
       <property name="text">
        <string>Browse…</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="rowSelVidFile" stretch="0,1,0">
     <item>
      <widget class="QLabel" name="headerVidFile">
       <property name="text">
        <string>Video File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fieldVidFileUrl">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPickVidFile">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Browse…</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="dlgBtnsTopPadding">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="rowDlgBtns" stretch="1,0">
     <item>
      <spacer name="dlgBtnsLeftPadding">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
